package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -503,6 +508,13 @@ def __init__(self, filename, lammps_coordinate_convention="auto",
                              " is not a valid option. "
                              f"Please choose one of {option_string}")
 
+        self._has_vels, self._has_forces = self._check_vels_forces()
+
+        if "unwrap_images" in kwargs:
```

Comment:
Please set a default value for the kwarg in `__init__` eg `unwrap_images=Flase` and then change this check.